### DISCUSSION TITLE
How can I get nitro.out.dir configuration in defineNitroPlugin?

### DISCUSSION THREAD

**Discussion by 275923543** [nuxt/nuxt#28958] [Questions]
// nuxt 3.13
export default defineNitroPlugin((nitro) => {
  // in here，How to obtain the value of nitro.out.dir
})

**Comment by xjccc** [comment]
Is `nitro.options` has the values?

**Reply by 275923543** [reply]
not have

**Reply by xjccc** [reply]
You could set a value in `runtimeConfig`?

**Reply by 275923543** [reply]
thanks.  
I need to read nitro.output.dir/serverDir/publicDir,  not set a value in runtimeConfig.
